src/Normalizer/FieldItemNormalizer.php

```diff
@@ -132,15 +132,15 @@ protected function constructValue($data, $context) {
    * entity. This is the reason for using target_instances, from which the
    * property path can be traversed up to the root.
    *
-   * @param \Drupal\Core\Field\FieldItemInterface $field_item
+   * @param \Drupal\Core\Field\FieldItemInterface $item
    *   The untranslated field item instance.
-   * @param $langcode
+   * @param string $langcode
    *   The langcode.
    *
    * @return \Drupal\Core\Field\FieldItemInterface
    *   The translated field item instance.
    */
-  protected function createTranslatedInstance(FieldItemInterface $item, $langcode) {
+  protected function createTranslatedInstance(FieldItemInterface $item, string $langcode) {
```

Comment:
Type hinting on primitive datatypes is not supported in PHP 5
http://php.net/manual/en/functions.arguments.php#functions.arguments.type-declaration